### ISSUE TITLE
No longer reporting in console location of documents with issues

### ISSUE THREAD

**Issue by aryzing** [dotansimha/graphql-code-generator#4227]
**Describe the bug**
When a document has an issue, eg. querying a field that doesn't exist, the tool would previously report the location of the offending file. Now it just reports the error, and I have no idea which document the error is in.

**To Reproduce**
Run `yarn generate` in the [reproduction repo in sandbox for this issue](https://codesandbox.io/s/brave-spence-smcvq?file=/document.graphql:81-95).

<!--
#############
#############
PLEASE READ THE FOLLOWING:
You can use this template to create a live sandbox of the issue: 
https://codesandbox.io/s/github/dotansimha/graphql-code-generator-issue-sandbox-template
Make sure sure fork this template, and run `yarn generate` in the terminal. 
Please make sure the Codegen and plugins version under `package.json` matches yours. 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~ ISSUES WITH A LIVE REPRODUCTION WILL BE ANSWERED QUICKLY ~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#############
#############
-->

1. My GraphQL schema:

```graphql
type Query {
    user(id: ID!): User!
}

type User {
    id: ID!
    username: String!
    email: String!
}
```

2. My GraphQL operations:

```graphql
query user {
    user(id: 1) {
        id
        username
        email
        i_do_not_exist
    }
}
```

3. My `codegen.yml` config file:

```yml
schema: schema.graphql
documents: document.graphql
generates:
  types.ts:
    plugins:
      - typescript
      - typescript-operations
```

**Expected behavior**
The errors shown in the console when running `yarn generate` in the [reproduction repo in sandbox for this issue](https://codesandbox.io/s/brave-spence-smcvq?file=/document.graphql:81-95) should inidcate wich file is responsible for the error.

**Environment:**

- OS: Ubuntu 18.04 LTS
- `@graphql-codegen/*`: 1.15.3
- NodeJS: v13.7.0

**Comment by dotansimha** [comment]
@ardatan It seems like an issue with `graphql-tools`. Can you please take a look? 
In the example @aryzing shared, it looks like the issue is coming from the `checkValidationErrors` - this method swallows the error details. In the original `GraphQLError` it seems like it exists:

```
[
  Error [GraphQLDocumentError]: Cannot query field "i_do_not_exist" on type "User".
      at document.graphql:6:9 {
    name: 'GraphQLDocumentError',
    message: 'GraphQLDocumentError: Cannot query field "i_do_not_exist" on type "User".'
  }
]
```

**Comment by dotansimha** [comment]
Fixed in v1.15.4

**Comment by ferm10n** [comment]
I found this issue after experiencing the same problem, but I don't think it's a regression. I discovered that it only happens if I use the `import-types` preset (2.1.8). I can't create a repro right now but I wanted to make a note here for anyone else who finds this.

**Comment by wmyers** [comment]
I get these kind of errors when a schema definition has the wrong function arguments.

For example :
```
type Mutation {
    deleteSomething(DeleteSomethingInput!): Boolean!
}
```
instead of:
```
type Mutation {
    deleteSomething(input: DeleteSomethingInput!): Boolean!
}
```

This happened when I used Copilot to try and generate a schema from a UML diagram - it wasn't bad but unsurprsingly there were a few syntax errors.

I then got a codegen error that gave a clue to the cause of the issue, but was swallowed enough not to know where it was, and no clues in the callstack either:
```
[FAILED]         Syntax Error: Expected ":", found "!".
[FAILED]         GraphQLError: Syntax Error: Expected ":", found "!".
[FAILED]     at syntaxError (/Users/wmyers/TRN/trnw-roadmap/apps/backend/node_modules/graphql/error/syntaxError.js:15:10)
etc
```

So annoyingly any time I saved on generating a schema was lost by having to step carefully through the code tracking down the error. If an app-level file and line number could be provided in the callstack that would be great.
